src/ResourceManager/HDInsight/Commands.HDInsight.Test/UnitTests/ConfigurationTests.cs

```diff
@@ -51,7 +51,7 @@ public void CanCreateNewConfig()
                     f.WriteObject(
                         It.Is<AzureHDInsightConfig>(
                             c =>
-                                c.ClusterType == HDInsightClusterType.Hadoop &&
+                                c.ClusterType == "Hadoop" &&
```

Comment:
Please use constant string instead of putting "Hadoop" string everywhere. you should also consider making a case insensitive comparison especially if this is something that customer may provide as parameter.